rfcs/0000-rfc-template.md

```diff
@@ -23,6 +24,13 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 
 ## Fields
 
+The `rule` fields being proposed are as follows:
+
+Field | Type | Description /Usage
+-- | -- | -- | -- | -- 
+rule.tags | array | Used to track the set of tags applied to a rule | Customers can use it to indicate: author, benchmark partial name, rule number, rule category etc. It will be useful when we extend the capability to add more rules
+rule.remediation | array | Used to capture remediation instructions that come from the benchmark / framework the rule is from 
```

Comment:
How would this value be used as an `array`? From the description, I'd expect a block of text with the instructions?

Reply:
There is possibility to have multiple remediation options for same misconfiguration, hence an array makes more sense. For example, we provide remediation manual steps, cli commands for CSPM misconfigurations.

Reply:
@kfirpeled @maxcold thoughts on this?

Reply:
Right now it's just a markdown text for our native integration, for Wiz it's also just a string. Unless we want to have a more complex structure for each remediation instruction, `text` should be enough as a type in my opinion.

Reply:
> There is possibility to have multiple remediation options for same misconfiguration, hence an array makes more sense. For example, we provide remediation manual steps, cli commands for CSPM misconfigurations.

I'm not sure I followed, currently, we don't have any examples of multiple remediation instructions. Can you please provide one?